### ISSUE TITLE
<it> field

### ISSUE THREAD

**Issue by elsdvlee** [kg-construct/rml-lv#9]
Should the `<it>` field result in a logical iteration that you can further reference?
What would the `<it>` contain in case of CSV of relational table? 
What about a joined logicalView? How would the `<it>` look like then?
Is the `<it>` field still needed?

**Comment by elsdvlee** [comment]
@pmaria What did you have in mind? 
Last useful note from the TF meetings on this topic say: 

> `<it>` is about accessing the raw value of an iteration and doesn't need to be referenceable
> basic need: to define what it is that the value of fields provides

**Comment by pmaria** [comment]
@elsdvlee 

I don't believe the `<it>` field is necessary anymore in the way we have currently defined the workings of fields. In the initial idea of fields a logical source record needed to have a string representation to work on. The string representation of the logical source was the `<it>` value. In the current definition this is not necessary anymore.

We might need to clarify what happens when you change reference formulations for a field. So a fields parent value was generated using a reference in a reference formulation X, what needs to happen with, or hold for, this value when the fields reference formulation is Y.
